spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/config/H2ServerConfigurationTests.java

```diff
@@ -112,16 +110,6 @@ void serverDoesNotStartWhenUrlIsH2ButInvalidForm() {
 						.hasMessage("Port value out of range: 99999"));
 	}
 
-	@Test
```

Comment:
**[UNRELATED]** This is not about adding/removing h2 but rather removing a test that does not work on all systems. This fails when run on my mac os. This test is really testing H2Server.tcpServer creation which should be heavily tested already by H2. Removing as we have no logic in place to check reserved ports.